src/hyperlight_guest/src/host_functions.rs

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2025 The Hyperlight Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+use alloc::format;
+use alloc::string::ToString;
+use alloc::vec::Vec;
+use core::slice::from_raw_parts;
+
+use hyperlight_common::flatbuffer_wrappers::function_call::FunctionCall;
+use hyperlight_common::flatbuffer_wrappers::function_types::ParameterType;
+use hyperlight_common::flatbuffer_wrappers::guest_error::ErrorCode;
+use hyperlight_common::flatbuffer_wrappers::host_function_details::HostFunctionDetails;
+
+use crate::error::{HyperlightGuestError, Result};
+use crate::P_PEB;
```

Comment:
P_PEB doesn't exist anymore.

Reply:
This whole module shouldn't be here. Removed.